### PR TITLE
Biconomy integration

### PR DESCRIPTION
**Pull request by nerfZael**

### CODE REVIEW

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token 2

**Review by dOrgJelli — APPROVED** [review]
Incredible!

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token 2

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token/send 3

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token/test_swap.py 2

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token/test_swap.py 3

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token/send 3

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token/send,agents/token/test_swap.py 3

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token/send,agents/token/test_swap.py 3

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token/test_swap_and_send.py 3

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token/research 2

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token/research/test_advanced.py::test_research_and_swap_many_tokens_subjective_simple,agents/token/research/test_advanced.py::test_research_and_swap.py::test_research_and_buy_multiple,agents/token/research/test_advanced.py::test_research_swap_and_send.py::test_research_buy_multiple_send_multiple,agents/token/research/test_advanced.py::test_research_swap_and_send.py::test_research_swap_and_send.py::test_research_buy_one_send_one 2

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token/research 3

File: smart_account_api/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+  const response = await smartAccount.deploy();
+
+  const receipt = await response.wait();
+  
+  res.json(receipt.receipt.transactionHash)
+}));
+
+app.post("/api/v1/account/transactions", handleError(async (req: Request, res: Response, next: NextFunction) => {
+  const chainId = parseInt(req.query.chainId as string);
```
Comment (cbrzn) [inline smart_account_api/src/index.ts:66]: nit: should we validate that `chainId` is indeed present in the `query` object?

**Pull request merged**